kafe2/fit/tools/contours_profiler.py

```diff
@@ -434,9 +471,10 @@ def plot_profile(
         show_fit_minimum=True,
         show_error_span=True,
         show_ticks=True,
-        label_ticks_in_sigma=True,
+        label_ticks="sigma",
         label_fit_minimum=True,
         font_scale=1.0,
+        use_fractions=False,
```

Comment:
I think it would make more sense to instead add an argument for the sigma interval for the ticks, with a default of 1.0.